tomls/omnibus-arbitrum-mainnet/perps/markets/btc.toml

```diff
@@ -0,0 +1,144 @@
+[var.perps_btc_market_settings]
+btc_perps_market_id = "200"
+btc_perps_market_skew_scale = "<%= String(35_000) %>"
+btc_perps_market_max_funding_velocity = "9"
+btc_perps_maker_fee_ratio = "0"
+btc_perps_taker_fee_ratio = "0.0005"
+btc_perps_max_market_size = "1200"
+btc_perps_max_market_value = "<%= String(50_000_000) %>"
+btc_perps_initial_margin_ratio = "0.744"
+btc_perps_maintenance_margin_scalar = "0.333"
+btc_perps_minimum_initial_margin_ratio = "0.02"
+btc_perps_flag_reward_ratio_d18 = "0.0003"
+btc_perps_max_liquidation_limit_accumulation_multiplier = "1.5"
+btc_perps_max_seconds_in_liquidation_window = "30"
+btc_perps_minimum_position_margin = "50"
+btc_perps_locked_oi_ratio = "0.25"
+btc_perps_max_liquidation_pd = "0.0005"
+btc_perps_endorsed_liquidator = "0x95A61Fa7454CA5f6A3CE01724e306Cd14a22D306"
+
+[invoke.createBtcPerpsMarket]
+target = ["perpsFactory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "createMarket"
+args = ["<%= settings.btc_perps_market_id %>", "Bitcoin", "BTC"]
+depends = ["var.perps_btc_market_settings"]
+
+[invoke.setPerpsPriceBtc]
+target = ["perpsFactory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "updatePriceData"
+args = [
+    "<%= settings.btc_perps_market_id %>",
+    "<%= extras.btc_oracle_id %>",
+    "<%= settings.bigCapStrictStalenessTolerance %>",
```

Comment:
This gets set to 0. I'm not sure if this is intentional.

Reply:
Fixing this on a fork of Sepolia unblocked me from an issue where wrapping never completed, but it needed a 0 second staleness tolerance. This should be updated for both networks.